spring-app/src/main/java/pl/edu/agh/cqm/data/model/ConfigCdn.java

```diff
@@ -0,0 +1,21 @@
+package pl.edu.agh.cqm.data.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Data
+@NoArgsConstructor
+@Entity
+@Builder
+@AllArgsConstructor
+@Table
+public class ConfigCdn {
+
+    @Id
+    @Column(nullable = false, updatable = false)
```

Comment:
String columns should have `length` parameter specified, otherwise they are stored as `TEXT` type in the database